llvm/test/Transforms/Attributor/call-simplify-pointer-info.ll

```diff
@@ -125,8 +131,10 @@ define i8 @call_simplifiable_3() {
 ; TUNIT-LABEL: define {{[^@]+}}@call_simplifiable_3
 ; TUNIT-SAME: () #[[ATTR0]] {
 ; TUNIT-NEXT:  entry:
-; TUNIT-NEXT:    [[BYTES:%.*]] = alloca [1024 x i8], align 16
-; TUNIT-NEXT:    [[I2:%.*]] = getelementptr inbounds [1024 x i8], ptr [[BYTES]], i64 0, i64 2
+; TUNIT-NEXT:    [[BYTES1:%.*]] = alloca [1 x i8], align 1
+; TUNIT-NEXT:    [[I2:%.*]] = getelementptr inbounds [1024 x i8], ptr [[BYTES1]], i64 0, i64 2
```

Comment:
Does this conform with the `inbounds` rule?

Reply:
that's a good point let me look into it, this might create a poison value...

Reply:
would it make sense to remove inbounds for such GEPs ? as the resultant address should be valid in that case.

Reply:
EDIT: I'll work on modifying that actual GEP instead of adding a new one.

Reply:
Update: I ended up changing the GEPs to use the correct type when generating pointers from the changed allocation.